resources/css/main.css

```diff
@@ -37,7 +37,14 @@ body {
     font-family: "Liberation Sans", "Helvetica Neue", "Helvetica", Arial, sans-serif;
     font-size: 16px;
 }
-
+body {
```

Comment:
What new functionality is this introducing?

Reply:
This feels like an abuse of CSS, since this means the document overflows the body.

Reply:
On very short pages, the footer isn't at the bottom of the screen and shows naked blue background. This was previously handled by hardcoding a pixel value and tuning it in JS, but flex allows for a pure CSS method.
Just changed to min-height to not overflow the body.